pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -10,20 +10,19 @@ let
     src = fetchFromGitHub {
       owner = "supermerill";
       repo = "SuperSlicer";
-      sha256 = "sha256-ThmsxFXI1uReK+JwpHrIWzHpBdIOP77kDjv+QaK+Azk=";
+      sha256 = "sha256-Q0gE0+A9WvwCCRt67W/MuF4lvFFlRNoyPwO4s+nr478=";
       rev = version;
+      fetchSubmodules = true;
     };
 
+    # We don't need PS overrides anymore, and gcode-viewer is embedded in the binary.
+    postInstall = "";
+
```

Comment:
```suggestion
```

Reply:
Ehm, we need to remove PS postInstall, or it would mess stuff up

Reply:
I didn't see that we are overwriting prusa-slicer here.

I still would prefer to overwrite postInstall with null instead of empty string if possible.

Reply:
Sure)

Reply:
@SuperSandro2000 ?